app/views/smart_answers/inputs/_value_question.html.erb

```diff
@@ -1,10 +1,11 @@
+<span class="govuk-caption-l"><%= @presenter.title %></span>
```

Comment:
Would it be possible to move this span to `/questions.html.erb` so it common for all questions and DRY's up the code?

Reply:
My plan is to move everything to the component so that there is always consistency and new styles don't cause it to break (visually)